### PR TITLE
[FEAT]: DARK THEME ADDED TO BOOKBRAINZ WEBSITE

### PR DESCRIPTION
**Pull request by Thuraabtech**
[FEAT]: DARK MODE THEME ADDED TO BOOKBRAINZ WEBSITE
### PROBLEM
THE BOOKBRAINZ WEBSITE DIDN'T HAVE THE DARK THEME MODE

### BB-611
ADDED A FEATURE TO THE BOOKBRAINZ WEBSITE FOR DARK THEME MODE 
WEBSITE ON DARK THEME MODE
Screenshot 2023-09-10 at 7 51 56 PM

WEBSITE ON LIGHT THEME MODE
Screenshot 2023-09-10 at 7 51 40 PM

### Areas of Impact
THE /bookbrainz-site/src/client/stylesheets/style.scss code has been modified

### CODE REVIEW

**Review by MonkeyDo — CHANGES_REQUESTED** [review]
Thank for opening your first PR! :)

I see in the screenshots that you have not fully set up your local environment (the error "role postgres doesn't exist" means you haven't [initialized your database](https://bookbrainz-dev-docs.readthedocs.io/en/latest/docs/installation.html#database-set-up)).

I would not be confident of this PR if you have only tested how it looks on the error page !
Please  finish your local set up and make sure you test all the pages.

The styles also seem much, much too simple.
For example in your screenshot, the navbar is still a bright white background.
I assume you will find many such issues once you test each page.

File: src/client/stylesheets/style.scss [hunk 1]
```diff
@@ -865,6 +865,24 @@ div[class~=collapsing]+div[class=card-header] .accordion-arrow {
 	margin: 10px;
 	max-width: 200px;
 }
+<<<<<<< HEAD
```
[the remark this answers is not in the document]
Comment (Thuraabtech) [reply src/client/stylesheets/style.scss:868]: thanks for pointing out the mistake , will definately look at the conflict . thats what i was thinking how did this come in the code , got to learn somehing new

**Comment by MonkeyDo** [comment]
Closing this for inactivity, and considering most of the work is missing.
Please feel free to reopen if and when you have made progress on the issue.

I'm here to help if you have questions or want help getting set up !

**Comment by Thuraabtech** [comment]
Yeah sure will work on it, I just wanna know where is the main js file

**Comment by MonkeyDo** [comment]
There is no main js file per se.
It's a big project with many pages, with server code in src/server and client-side code in src/client

**Pull request closed without merging**